### PR TITLE
refactor: consistent name for room

### LINKED ISSUE #592: [OTHER] keep consistant naming for chat rooms
### What would you like to share?

The naming for chat rooms are inconsistent in the backend and frontend

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Dun-sin**
# Fixes Issue

**My PR closes #592 **

# 👨‍💻 Changes proposed (What did you do ?)
- changed naming to be consistent

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

**Comment by Dun-sin** [comment]
@mathiasayivor please review

File: src/server/models/ActiveUserModel.ts [hunk 1]
```diff
@@ -11,10 +12,10 @@ export interface ActiveUserSchemaDocument extends Document {
     id: string;
     email: string | null;
     loginId: string;
-    socketConnections: any[];
-    socketIds: any[];
-    currentChatId: string | null;
-    chatIds: any[];
+    socketConnections: Socket[];
+    socketIds: string[];
+    currentChat: string | null;
```
Comment (mathiasayivor) [inline src/server/models/ActiveUserModel.ts:17]: Why use `currentChat` here, but `currentroom` in src/types/types.ts (ActiveUserType)?

File: src/pages/searching.tsx [hunk 2]
```diff
@@ -96,15 +96,23 @@ const Searching = () => {
...
+    router.push('/anonymous');
+  }, [app.currentroom]);
+
+  useEffect(() => {
+    if (!app.currentroom) {
+      return;
+    }
+
+    router.push('/anonymous');
+  }, [app.currentroom]);
```
Comment (mathiasayivor) [inline src/pages/searching.tsx:115]: This is a repitition of line (101 - 107)

**Review by mathiasayivor — CHANGES_REQUESTED** [review]
`room` and `currentroom` is a bit confusing; When you first see, you may think `room` is an object representing a room, but it's a string, same thing applies to `currentroom`.

Can we use `roomId`/`currentRoomId` instead?

File: src/server/models/ActiveUserModel.ts (hunk 1 above)
Comment (Dun-sin) [reply src/server/models/ActiveUserModel.ts]: doesn't really affect it much, but i still changed it either way

File: src/pages/searching.tsx (hunk 2 above)
Comment (Dun-sin) [reply src/pages/searching.tsx]: thanks didn't notice while merging

File: src/server/models/ActiveUserModel.ts (hunk 1 above)
Comment (mathiasayivor) [reply src/server/models/ActiveUserModel.ts]: Ok. But I think it introduces inconsistency

**Review by mathiasayivor — APPROVED** [review]

**Comment by mathiasayivor** [comment]
LGTM

File: src/server/models/ActiveUserModel.ts (hunk 1 above)
Comment (Dun-sin) [reply src/server/models/ActiveUserModel.ts]: i agree, i was thinking too far on that one

**Pull request merged**